config/grafana/values.yaml

```diff
@@ -7,4 +7,13 @@ adminPassword: admin
 persistence:
   type: pvc
   enabled: true
-  storageClassName: standard-v2
\ No newline at end of file
+  storageClassName: standard-v2
+
+datasources:
+  datasources.yaml:
+    apiVersion: 1
+    datasources:
+    - name: Prometheus
+      type: prometheus
+      url: http://thanos-query-frontend.{{ template "grafana.namespace" . }}.svc.cluster.local:9090
```

Comment:
This should be available only for superusers... Are we sure that the tenant users will not have access to it with this?

Reply:
this is only for PoC.
tenant users will have their own organization without this datasource

Reply:
Can we somehow configure the permission of the datasource to make sure that only admin can see this datasource?

Reply:
as I said, this datasource will be in a different organization. for tenant we will be creating own organization